src/test/java/mServer/test/ZDFClientMock.java

```diff
@@ -22,9 +22,11 @@ public ZDFClient get() {
         return mock;
     }
     
-    public void setUp(String url, String jsonFile) throws IOException, URISyntaxException {
-        WebResource mockWebResource = Mockito.mock(WebResource.class);
-        when(mock.createResource(url)).thenReturn(mockWebResource);
-        when(mock.execute(mockWebResource)).thenReturn(JsonFileReader.readJson(jsonFile));
+    public void setUp(String url, String jsonFile) {
+        try {
+        when(mock.execute(url)).thenReturn(JsonFileReader.readJson(jsonFile));
+        } catch(IOException | URISyntaxException ex) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)